src/devices/cpu/gigatron/gigatron.cpp

```diff
@@ -25,9 +25,11 @@ DEFINE_DEVICE_TYPE(GTRON, gigatron_cpu_device, "gigatron_cpu", "Gigatron CPU Dev
 #endif
 
 
-#define gigatron_readop(A) m_program->read_dword(A)
+#define gigatron_readop(A) m_program->read_byte(A)
```

Comment:
You want to use `read_word` instead of `read_byte` here, given that you're storing the instruction in a 16-bit variable (and a 16-bit space with a -1 shift doesn't allow access to individual bytes anyway).